src/AccessibilityInsights.SharedUx/Properties/AutomationIDs.cs

```diff
@@ -13,5 +13,15 @@ public static class AutomationIDs
         public static string TelemetryDialogExitButton { get; } = nameof(TelemetryDialogExitButton);
```

Comment:
John and I will continue with this AutomationIDs strategy and reassess later in the feature whether we like it, want to group better, etc